examples/cli/max_depth.py

```diff
@@ -1,3 +1,23 @@
+"""
+This model script contains a model, pytorch_model, which has
+two sub-modules, fc and fc2. You can use it to experiment with
+the --max-depth option, which analyzes/builds/benchmarks sub-modules
+of any modules in the top-level script.
```

Comment:
Overall, this MR looks good for now.

In the future, we might want to have a quick tutorial in which each example file can be used for more than one thing. At that point, we can probably rename `max_depth.py `to `pytorch_linear.py`, `keras.py` to `keras_linear.py` and then use `pytorch_linear.py` as both a hello world example as well as a `max_depth` example and so on.

```
.
└── examples/
    ├── pytorch_linear.py
    ├── keras_linear.py
    └── readme.md (tutorial)
```

Reply:
#23 is meant to be a comprehensive look at examples. Linking your comment there.